package.json

```diff
@@ -12,11 +12,12 @@
   "dependencies": {
     "@medusajs/medusa": "^1.6.0",
     "@medusajs/medusa-cli": "^1.3.3",
+    "@medusajs/cache-inmemory": "next",
```

Comment:
question/todo: I suppose we are adding it to support the next medusa version, should we add the module definition in the medusa config?

Reply:
The in-memory cache module is a default module, so we won't need to add it to `medusa-config.js`.